mlir/utils/jenkins/Jenkinsfile

```diff
@@ -1057,5 +1058,68 @@ pipeline {
                 }
             }
         }
+
+        stage ("Code coverage") {
+            when {
+                beforeAgent true;
+                equals expected: true, actual: params.runCodeCoverage;
+            }
+            matrix {
+                axes {
+                    axis {
+                        name 'CHIP'
+                        values 'gfx908', 'gfx90a'    //, 'gfx1030', 'gfx1100'
+                    }
+                }
+                agent {
+                    docker {
+                        image dockerImage()
+                        args dockerArgs()
+                        label getLabelFromChip("${CHIP}")
+                        alwaysPull true
+                    }
+                }
+                environment {
+                    PATH="/opt/rocm/llvm/bin:$PATH"
+                    HOME="${WORKSPACE}"
+                    // Note the %m to avoid issues with threads and dynamic libraries.
+                    LLVM_PROFILE_FILE="${WORKSPACE}/build/%m-%p.profraw"
```

Comment:
The %m fills in a unique identifier for the binary;  %p is the process ID.  Otherwise, the default file is "default.profraw" and we get collisions from multiple simultaneous tests and also when both main program and .so are instrumented.

Reply:
Also, by sticking them all in build, the *.profraw files are easier to find.